app/src/main/java/io/apicurio/registry/metrics/RestMetricsResponseFilter.java

```diff
@@ -0,0 +1,61 @@
+package io.apicurio.registry.metrics;
+
+import static io.apicurio.registry.metrics.MetricIDs.*;
+
+import static org.eclipse.microprofile.metrics.MetricRegistry.Type.APPLICATION;
+import static org.eclipse.microprofile.metrics.MetricType.COUNTER;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.ext.Provider;
+
+import org.eclipse.microprofile.metrics.Counter;
+import org.eclipse.microprofile.metrics.Metadata;
+import org.eclipse.microprofile.metrics.MetricRegistry;
+import org.eclipse.microprofile.metrics.Tag;
+import org.eclipse.microprofile.metrics.annotation.RegistryType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Provider
+@RestMetricsResponseFilteredNameBinding
+public class RestMetricsResponseFilter implements ContainerResponseFilter {
```

Comment:
I would replace the class name by something more meaningful, maybe `RegisterStatusCodeResponseFilter`.

Reply:
Unless we think the filter is going to track additional metrics.  But if we're renaming I'd suggest `RestStatusCodeMetricsResponseFilter`.

Reply:
I think we should keep the current name as there is other information other than status code being gathered